dvc/output.py

```diff
@@ -708,22 +708,20 @@ def verify_metric(self):
             raise DvcException(
                 f"verify metric is not supported for {self.protocol}"
             )
-
-        if not self.metric or self.plot:
+        if not self.metric:
             return
 
         if not os.path.exists(self.fs_path):
             return
 
-        name = "metrics" if self.metric else "plot"
         if os.path.isdir(self.fs_path):
-            msg = "directory '%s' cannot be used as %s."
-            logger.debug(msg, str(self), name)
+            logger.debug(f"directory '{self}' cannot be used as metrics.")
```

Comment:
I worked hard on https://github.com/iterative/dvc/pull/7961, don't start undoing it!

Reply:
Ah, sorry @alexmojaki, I missed that one. 
@ykasimov can you roll the log calls back?

Reply:
reverted in #8081